packages/components/pages/f-contact-preferences/src/components/ContactPreferences.vue

```diff
@@ -177,6 +169,28 @@ export default {
             'editPreference'
         ]),
 
+
+
+        /**
+        * Returns translation string if it exists
+        * @param {string} key - The key of the preference that needs changing
+        */
+        getEmailDescription (key) {
```

Comment:
Just to clarify for my understanding more than anything – I'm guessing this is here to help with error handling, as passing the direct value if it's not defined causes issues?

Reply:
Yes, if we don't perform the check a string is returned  = `news.smsDescription`. We are also wrapping the string in brackets (existing functionality)